### PR TITLE
fix(ci): fix iceberg-cdc integration_tests

### PR DESCRIPTION
**Pull request by ZENOTME**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Move python check to docker.

fix: https://buildkite.com/risingwavelabs/integration-tests/builds/316#018b6723-fddd-4243-a107-d3d56cacd716

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Comment by chenzl25** [comment]
> #### ⚠️ GitGuardian has uncovered 1 secret following the scan of your pull request.
> Please consider investigating the findings and remediating the incidents. Failure to do so may lead to compromising the associated services or software components.
> 
> 🔎 Detected hardcoded secret in your pull request
> 🛠 Guidelines to remediate hardcoded secrets
> 🦉 [GitGuardian](https://dashboard.gitguardian.com/auth/login/?utm_medium=checkruns&utm_source=github&utm_campaign=cr1) detects secrets in your source code to help developers and security teams secure the modern development process. You are seeing this because you or someone else with access to this repository has authorized GitGuardian to scan your pull request.Our GitHub checks need improvements? [Share your feedbacks](https://050127dayho.typeform.com/to/KmeAPTMk)!


Could you please remove the password from iceberg-cdc run_test.sh?

File: integration_tests/iceberg-cdc/python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# 使用 Python 3.8 作为基础镜像
```
Comment (chenzl25) [inline integration_tests/iceberg-cdc/python/Dockerfile:1]: Remove the comments as well

**Comment by liurenjie1024** [comment]
LGTM, as @neverchanje says, please have a try with buildkite before merging.

**Comment by ZENOTME** [comment]
Test in ci:
https://buildkite.com/risingwavelabs/integration-tests/builds/322#018b6f1f-96fa-491d-9343-be2f774c1726

**Review by liurenjie1024 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**